extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -302,37 +471,101 @@ export async function newDeployment(
     return newCredentialForced(state) || newCredentialSelected(state);
   };
 
-  const hasMultipleEntryPoints = () => {
+  const hasMultiplePossibleEntryPointFiles = () => {
     return entryPointListItems.length > 1;
   };
```

Comment:
This is still using `entryPointListItems` rather than `discoveredEntryPoints` so it returns `true` since the "Open Files" text is an item in the `entryPointListItems`.

This causes the steps to be incorrect when using the "Add New Deployment" in the Home View. This doesn't occur when using the entrypoint button since it doesn't use the "Open Files" bit.